arduino-ide-extension/src/browser/survey/survey-retriever.ts

```diff
@@ -0,0 +1,20 @@
+import { injectable } from '@theia/core/shared/inversify';
+
+export type Survey = {
+  url: URL;
+  id: string;
+};
+
+const SURVEY_BASE_URL = 'https://surveys.hotjar.com/';
+const surveyId = '17887b40-e1f0-4bd6-b9f0-a37f229ccd8b';
```

Comment:
Why did you make the `getSurvey` implementation parameterized if these constants are hard-coded?